src/org/zaproxy/zap/extension/pscan/PassiveScannerOptionsPanel.java

```diff
@@ -42,27 +44,22 @@
     private static final long serialVersionUID = 1L;
 
     private final JCheckBox scanOnlyInScopeCheckBox;
+    private final JCheckBox scanFuzzerMessagesCheckBox;
 
     public PassiveScannerOptionsPanel(I18N messages) {
         setName(messages.getString("pscan.options.main.name"));
 
-        GroupLayout layout = new GroupLayout(this);
-        setLayout(layout);
-
-        layout.setAutoCreateGaps(true);
-        layout.setAutoCreateContainerGaps(true);
-
-        scanOnlyInScopeCheckBox = new JCheckBox();
-        JLabel scanOnlyInScopeLabel = new JLabel(messages.getString("pscan.options.main.label.scanOnlyInScope"));
-        scanOnlyInScopeLabel.setLabelFor(scanOnlyInScopeCheckBox);
-
-        layout.setHorizontalGroup(
-                layout.createSequentialGroup().addComponent(scanOnlyInScopeLabel).addComponent(scanOnlyInScopeCheckBox));

```

Comment:
Replace with a `createParallelGroup` (and add `scanOnlyInScopeCheckBox` and `scanFuzzerMessagesCheckBox`).
Same for the vertical group but with a `createSequentialGroup`.